addins/Cake.AutoRest.yml

```diff
@@ -2,9 +2,10 @@ Name: Cake.AutoRest
 NuGet: Cake.AutoRest
 Assemblies:
 - "/**/Cake.AutoRest.dll"
-Repository: https://github.com/agc93/Cake.AutoRest
+Repository: https://github.com/agc93/Cake.AutoRest/
```

Comment:
@Jericho wondering if this is diff "enough" to warrant an update?

Reply:
I mentioned this to you earlier, maybe we should standardize: Urls in our YAML files should/should not end with a slash.